tests/unit/test_color_refinement_dorien.py

```diff
@@ -7,7 +7,7 @@
 from color_refinement import is_isomorphisms
 from graph_io import *
 
-PATH = 'graphs/colorref'
+PATH = 'graphs/colorref'# to run locally from PyCharm: PATH = '../../graphs/colorref'
```

Comment:
Dit, of je zet in je Run Config de working directory op de root i.p.v. `./tests/unit/`